var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -609,6 +610,9 @@ def nvariant_bool(feature):
         if "+tbb" in spec:
             cmake_args.append("-DVTK_SMP_IMPLEMENTATION_TYPE=TBB")
 
+        if "+nvindex" in spec:
+            cmake_args.append("-DPARAVIEW_PLUGIN_ENABLE_pvNVIDIAIndeX:BOOL=ON")
+
```

Comment:
For future reference: There are helpers that could simplify these variant checks (see docs at https://spack.readthedocs.io/en/latest/build_systems/cmakepackage.html#adding-flags-to-cmake).